Pipfile

```diff
@@ -26,7 +26,10 @@ requestsdefaulter = "*"
 pycryptodome = "*"
 flask-talisman = "*"
 email_validator = "*"
-google-cloud-pubsub = "==1.7.0"
+google-cloud-pubsub = "*"
+google-cloud-storage = "*"
+python-gnupg = "==0.4.7"
```

Comment:
why? this is the package which should get continuous updates. may be lets unpin it in another card.